harness/determined/cli/trial.py

```diff
@@ -181,7 +182,17 @@ def download(args: Namespace) -> None:
             raise ValueError(f"No checkpoints found for trial {args.trial_id}")
         checkpoint = checkpoints[0]
 
-    path = checkpoint.download(path=args.output_dir)
+    try:
+        path = checkpoint.download(path=args.output_dir)
+    except errors.CheckpointStateException:
+        print(
+            f"Invalid Checkpoint State for checkpoint {checkpoint.uuid}.\n",
+            "It's likely that this checkpoint is deleted. Please retry with a valid checkpoint UUID.",
+            sep=''
+        )
+        return
+    except Exception as e:
```

Comment:
why is this needed? unchecked exceptions (exceptions that are not `CheckpointStateException`) should be uncaught and raised automatically?

Reply:
Removed